common/djangoapps/third_party_auth/urls.py

```diff
@@ -2,11 +2,12 @@
 
 from django.conf.urls import include, patterns, url
 
-from .views import inactive_user_view, saml_metadata_view
+from .views import inactive_user_view, saml_metadata_view, lti_login_view
 
 urlpatterns = patterns(
     '',
     url(r'^auth/inactive', inactive_user_view),
     url(r'^auth/saml/metadata.xml', saml_metadata_view),
+    url(r'^auth/login/(?P<backend>lti)/$', lti_login_view),
```

Comment:
I think this is a bit of a tradeoff. On a technical basis, I agree with you; the login portion is just preparing to immediately complete. However, from a usability perspective, I wonder if it makes more sense for tool consumer administrators to type in a URL that contains "login" in it?